dependencies/nlohmann/meson.build

```diff
@@ -0,0 +1,8 @@
+# nlohmann Dependency File
```

Comment:
Can I suggest changing it to `nlohmann_json` in all?

Reply:
in the comments?

Reply:
Yes, comments and filename too. 
Nlohmann is the name of programmer, `nlohmann_json` is the JSON library he created.

Reply:
changed 👍